repos.yaml

```diff
@@ -976,3 +976,8 @@ repositories:
     maintainers:
       - name: Sean Krail
         email: sean.krail@jitterbit.com
+  - name: ververica
+    url: https://charts.ververica.com
+    maintainers:
+      - name: Ververica
+        email: support@ververica.com
```

Comment:
@austince it doesn't appear you work for Ververica. Is there someone who works there who can vouch that they want this listed?

Reply:
No, we're a customer - I'll reach out to them and get approval.